score-lang-cli/src/main/java/org/openscore/lang/cli/services/ScoreServicesImpl.java

```diff
@@ -131,12 +143,43 @@ public synchronized void onEvent(ScoreEvent scoreEvent) throws InterruptedExcept
                         printWithColor(Ansi.Color.YELLOW, prefix + taskName);
                     }
                     break;
+                case ScoreLangConstants.EVENT_OUTPUT_END:
+                    if(quietListener){
```

Comment:
the quiet mode refers to not showing tasks that are executed. The flow outputs are printed even if quietListener is true. atm the implementation is: normal mode - print tasks (no outputs), quiet mode - print flow outputs (no tasks)
Once we merge the flow_outputs_to_cli branch the normal mode will print flow outputs by default and the condition won't be necessary.